DESCRIPTION

```diff
@@ -34,6 +34,7 @@ Imports:
     locfit,
     matrixStats,
     methods,
+    preprocessCore,
```

Comment:
I've moved preprocessCore to suggested and added requireNamespace to the quantile normalization function.

Reply:
I am getting the following warning during R CMD: 

```
> checking for unstated dependencies in examples ... WARNING
  'library' or 'require' calls not declared from:
    ‘c’ ‘preprocessCore’ ‘vcfR’
```
I tried to set these up like in your ZINB function since that also has a dependency suggestion, but I must have missed something.

Reply:
I don't think you can give a vector to `requireNamespace()` so you might have to do something like:

```r
if (requireNamespace("preprocessCore", quietly = TRUE) & requireNamespace("vcfR", quietly = TRUE)) {
...
}
```